src/adapters/scene-stream-access-manager.ts

```diff
@@ -0,0 +1,61 @@
+import { AppComponents, AddSceneStreamAccessInput, ISceneStreamAccessManager, SceneStreamAccess } from '../types'
+import SQL from 'sql-template-strings'
+
+export async function createSceneStreamAccessManagerComponent({
+  database,
+  logs
+}: Pick<AppComponents, 'database' | 'logs'>): Promise<ISceneStreamAccessManager> {
+  const logger = logs.getLogger('scene-stream-access-manager')
+
+  async function addAccess(input: AddSceneStreamAccessInput): Promise<void> {
+    logger.debug('Adding stream access', { place_id: input.place_id })
+
+    await database.query(
+      SQL`UPDATE scene_stream_access 
```

Comment:
I can't find the migration for the `scene_stream_access` table, would you mind including it in the PR?